### PR TITLE
bioin-1620 reference is weird inside parallel

### PR DESCRIPTION
**Pull request by mayalevy**
fixed the statistics log . Can be seen here at the bottom:

gsutil cat gs://cromwell-backend-ultima-data-307918/cromwell-execution/SVPipeline/760427b3-7ca6-468b-bed1-1bb468ce3eab/call-LongHomopolymersAlignmnet/LongHomopolymersAlignmnet.lo

### CODE REVIEW

File: scripts/align_long_homopolymers.py [hunk 1]
```diff
@@ -355,12 +355,7 @@ def realign_homopolymers(cram_path, output_path, reference_path, homopolymer_len
                         read.cigar = cigar_string_to_cigartuples(adjusted_cigar)
                     output.write(read)
 
-    logger.info(f"Total reads: {count}")
```
Comment (ilyasoifer) [inline scripts/align_long_homopolymers.py:358]: These prints are useful, why delete them?

Comment (mayalevy) [reply scripts/align_long_homopolymers.py:358]: @ilyasoifer they print per contig now, not the whole statistics. So I can add the contig name for that, but we have a lot of them, it prints pretty a lot of text. Do you what to keep it per contig?

Comment (ilyasoifer) [reply scripts/align_long_homopolymers.py:358]: Ah, got it! 
Maybe you could aggregate those and print once?

**Comment by ilyasoifer** [comment]
@shbenari - where should we merge this? Can we merge to the "master" of the gridss?

**Comment by shbenari** [comment]
@giuliovn where is the docker built from?
Do we build it on every PR?

**Comment by giuliovn** [comment]
> @giuliovn where is the docker built from? Do we build it on every PR?

no, it is started manually

**Comment by giuliovn** [comment]
> @shbenari - where should we merge this? Can we merge to the "master" of the gridss?

master should be ok

File: scripts/align_long_homopolymers.py (hunk 1 above)
Comment (mayalevy) [reply scripts/align_long_homopolymers.py:358]: Done

File: scripts/align_long_homopolymers.py [hunk 2]
```diff
@@ -540,12 +542,20 @@ def insert_operation_into_cigar(cigar, position, op_size, start_pos, op_type):
...
 
-    Parallel(n_jobs=args.n_jobs, max_nbytes=None)(
+    results = Parallel(n_jobs=args.n_jobs, backend="multiprocessing", max_nbytes=None)(
         delayed(realign_homopolymers)(
             args.input, f"{args.output}{contig}.bam", args.reference, args.homopolymer_length, contig
         )
         for contig in large_contigs
     )
+    total_count = 0
+    total_count_homopolymere = 0
```
Comment (ilyasoifer) [inline scripts/align_long_homopolymers.py]:
```suggestion
    total_count_homopolymer = 0
```

File: scripts/align_long_homopolymers.py [hunk 3]
```diff
@@ -540,12 +542,20 @@ def insert_operation_into_cigar(cigar, position, op_size, start_pos, op_type):
...
+    total_choices = [0, 0, 0, 0]
+    for count, count_homopolymere, choices in results:
```
Comment (ilyasoifer) [inline scripts/align_long_homopolymers.py]:
```suggestion
    for count, count_homopolymer, choices in results:
```

**Review by ilyasoifer — APPROVED** [review]

**Pull request merged**